### PR TITLE
ADEN-9388 Insert incontent player on start

### PR DESCRIPTION
**Pull request by preludek**
## Links

* http://wikia-inc.atlassian.net/browse/JIRA-123
* more relevant links

## Description

A description written using complete sentences of the changes being made and any context necessary to understand them.

## Reviewers

Use `@` mentions here. Alternatively, assign the pull request to a person.

### CODE REVIEW

File: app/modules/ads/slots-loader.js [hunk 1]
```diff
@@ -136,11 +137,24 @@ export const slotsLoader = {
     );
   },
 
-  injectFirstSlot(firstSlotName) {
+  loadIncontentPlayer() {
+    const { context } = window.Wikia.adEngine;
+
+    const slotName = 'incontent_player';
+    const isDisabledOnPush = context.get('slots.incontent_player.disablePushOnScroll');
```
Comment (bckatarzyna) [inline app/modules/ads/slots-loader.js]:
```suggestion
    const isDisabledOnScroll = context.get('slots.incontent_player.disablePushOnScroll');
```

File: app/modules/ads/slots-loader.js [hunk 2]
```diff
@@ -136,11 +137,24 @@ export const slotsLoader = {
...
+
+    this.handleBidsRefreshPromise(
+      this.injectFirstSlot,
+      slotName,
+      isDisabledOnPush,
```
Comment (bckatarzyna) [inline app/modules/ads/slots-loader.js]:
```suggestion
      isDisabledOnScroll,
```

File: app/modules/ads/slots-loader.js [hunk 3]
```diff
@@ -136,11 +137,24 @@ export const slotsLoader = {
...
+    );
+  },
+
+  injectFirstSlot(firstSlotName, disableSlotOnPush = null) {
```
Comment (bckatarzyna) [inline app/modules/ads/slots-loader.js]:
```suggestion
  injectFirstSlot(firstSlotName, disablePushOnScroll = false) {
```

File: app/modules/ads/slots-loader.js [hunk 4]
```diff
@@ -136,11 +137,24 @@ export const slotsLoader = {
...
     const { context, slotInjector, utils } = window.Wikia.adEngine;
 
     utils.logger(logGroup, `injection started: ${firstSlotName}`);
-    slotInjector.inject(firstSlotName);
+    slotInjector.inject(firstSlotName, disableSlotOnPush);
```
Comment (bckatarzyna) [inline app/modules/ads/slots-loader.js]:
```suggestion
    slotInjector.inject(firstSlotName, disablePushOnScroll);
```

File: app/modules/ads/slots-loader.js (hunk 3 above)
Comment (bckatarzyna) [reply app/modules/ads/slots-loader.js]: `disablePushOnScroll` is boolean type

Comment (bckatarzyna) [reply app/modules/ads/slots-loader.js]: Could you also change the function name to `injectSlot` as now it is used two times?

**Review by bckatarzyna — APPROVED** [review]

File: app/modules/ads/slots-loader.js (hunk 1 above)
Comment (preludek) [reply app/modules/ads/slots-loader.js]: 👍

File: app/modules/ads/slots-loader.js (hunk 2 above)
Comment (preludek) [reply app/modules/ads/slots-loader.js]: 👍

File: app/modules/ads/slots-loader.js (hunk 3 above)
Comment (preludek) [reply app/modules/ads/slots-loader.js]: 👍

File: app/modules/ads/slots-loader.js (hunk 4 above)
Comment (preludek) [reply app/modules/ads/slots-loader.js]: 👍

**Pull request merged**